person/person.go

```diff
@@ -8,69 +8,21 @@ import (
 const personIndexName = "person"
 
 type Person struct {
-	ID              *int64 `json:"id"`
-	UID             string `json:"uId"`
-	Normalizeduid   int64  `json:"normalizedUid"`
-	SageID          string `json:"sageId"`
-	CaseRecNumber   string `json:"caseRecNumber"`
-	Workphonenumber struct {
```

Comment:
It was confusing to see all of these fields that aren't actually indexed, so I've cleaned this type up to what we actually use